docs/csharp/language-reference/compiler-messages/cs0433.md

```diff
@@ -16,6 +16,16 @@ The type TypeName1 exists in both TypeName2 and TypeName3
   
  To resolve this error, use the alias feature of the ([**References**](../compiler-options/inputs.md#references)) compiler option or do not reference one of your assemblies.  
 
+## How to identify the conflicting assemblies
```

Comment:
Shouldn't this section go after line 32, otherwise lines 29-32 appear to be part of the "how to identify" section.